packages/module/patternfly-docs/content/extensions/react-log-viewer/demos/LogViewer.md

```diff
@@ -2,10 +2,9 @@
 id: Log viewer
 beta: true
 section: extensions
-source: react-demos
```

Comment:
I _think_ we want to keep the source metadata here so that the demos show up under a different tab from the examples.